packages/block-library/src/column/edit.js

```diff
@@ -21,13 +21,12 @@ import { __ } from '@wordpress/i18n';
 function ColumnEdit( {
 	attributes,
 	setAttributes,
-	className,
 	updateAlignment,
 	hasChildBlocks,
 } ) {
 	const { verticalAlignment, width } = attributes;
 
-	const classes = classnames( className, 'block-core-columns', {
```

Comment:
Seems like this prop also adds the custom className attribute, should we add it to the wrapper too?

Reply:
Oh, right, I forgot about that one. We can also add it automatically.

Reply:
Added it